src/App.tsx

```diff
@@ -1,57 +1,113 @@
-import React from 'react';
-import './App.scss';
+import React, { useCallback, useMemo, useState } from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
Still not fixed from the previous review
<img width="877" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/64750958/6c176dfc-7c1d-4b92-a8ba-0fe794323f50">